js-joda-timezone/build.boot

```diff
@@ -16,13 +16,16 @@
 
 (deftask package []
   (comp
-    (download :url (format "https://github.com/js-joda/js-joda-timezone/archive/v%s.zip" +lib-version+)
+    (download :url (format "https://github.com/js-joda/js-joda/archive/@js-joda/timezone@%s.zip" +lib-version+)
                 :unzip true)
     
-    (sift :move {#"^js-joda-timezone-([\d\.]+)/dist/js-joda-timezone\.js$" "cljsjs/js-joda-timezone/development/js-joda-timezone.inc.js"})
+    (sift :move {#"^js-joda--js-joda-timezone-([\d\.]+)/packages/timezone/dist/js-joda-timezone\.js$" "cljsjs/js-joda-timezone/development/js-joda-timezone.inc.js"})
     (minify :in  "cljsjs/js-joda-timezone/development/js-joda-timezone.inc.js"       
             :out "cljsjs/js-joda-timezone/production/js-joda-timezone.min.inc.js")
     (sift :include #{#"^cljsjs"})
-    (deps-cljs :no-externs true :name "cljsjs.js-joda-timezone" :requires ["cljsjs.js-joda"] )
+    (deps-cljs 
+      :no-externs true 
+      :provides ["@js-joda/timezone"]
+      :requires ["@js-joda/core"])
```

Comment:
Timezone doesn't export a object so global-exports isn't needeed?

Reply:
yes only @js-joda/core adds global-exports. timezone and locale just do :provides here

Reply:
sorry I should explain how it works, when timezone executes, it modifies the global JSJoda object - extending its capabilities